x/bank/atlas/atlas.toml

```diff
@@ -0,0 +1,34 @@
+[module]
+description = "The bank module is responsible for handling multi-asset coin transfers between accounts and tracking special-case pseudo-transfers which must work differently with particular kinds of accounts."
+homepage = "https://github.com/cosmos/cosmos-sdk"
+keywords = [
+  "tokens",
+  "bank",
+  "transfer",
+  "asset",
+  "fungible",
+  "coins",
+]
+
+name = "x/bank"
+
+[bug_tracker]
+url = "https://github.com/cosmos/cosmos-sdk/issues"
+
+[[authors]]
+name = "alexanderbez"
+
+[[authors]]
+name = "fedekunze"
+
+[[authors]]
+name = "cwgoes"
+
+[[authors]]
+name = "alessio"
```

Comment:
maybe late but what is the reasoning for having to specify multiple author sections?

Reply:
I don't follow. This is how you represent an array of objects in TOML.

Reply:
https://toml.io/en/v1.0.0-rc.3#array-of-tables

Reply:
I was thinking of a comma separated string but this is better.

Reply:
Yup -- that's also an option and how I think cargo does it too. I just chose this way because it's a bit more explicit. Nothing is really set in stone yet so if you think that approach is better, it's not too late to change it.